src/crypto/load.py

```diff
@@ -0,0 +1,40 @@
+import os
+import json
+from typing import Dict
+from datetime import datetime
+
+from google.cloud.storage import Client
+from google.cloud.storage.bucket import Bucket
+from google.oauth2 import service_account
+
+from crypto.utils.api import Connection
+from crypto.utils.constants import LISTINGS_LATEST_URL
+
+
+class Load:
+
+    def __init__(self) -> None:
+        self._service_account = os.getenv("CLOUD_STORAGE")
```

Comment:
could this be constructor arg to "inject" it? so we are not dependent on os environ only? might be handy when testing

Reply:
good call. I'll work on this.